### DISCUSSION TITLE
What is the source of Oracle HNT price?

### DISCUSSION THREAD

**Discussion by arunmp2004** [davetapley/helium-tax#48] [General]
What is the source of Oracle HNT price?
Does Helium maintains this data or is it coming from a specific exchange like Binance US?

Thanks,
Arun

**Comment by davetapley** [comment]
See here: https://docs.helium.com/blockchain/oracles/#who-are-the-hnt-price-oracles

**Comment by arunmp2004** [comment]
Thank you!

Best Regards,
Arun
